packages/ui/src/components/calendar/DatePresenter.svelte

```diff
@@ -26,11 +26,13 @@
   import DPCalendar from './icons/DPCalendar.svelte'
   import DPCalendarOver from './icons/DPCalendarOver.svelte'
   import { getMonthName } from './internal/DateUtils'
+  import { registerFocus } from '../../focus'
 
   export let value: number | null | undefined
   export let mode: DateRangeMode = DateRangeMode.DATE
   export let mondayStart: boolean = true
   export let editable: boolean = false
+  export let input: HTMLButtonElement | undefined = undefined
```

Comment:
should it be exported? do you expect it to be passed from outside of the component?